script/bin/operator.rs

```diff
@@ -485,9 +485,11 @@ where
 
         self.network_prover
             .prove(&self.pk, &stdin)
-            .strategy(FulfillmentStrategy::Reserved)
+            .strategy(FulfillmentStrategy::Auction)
             .skip_simulation(true)
             .cycle_limit(10_000_000_000)
+            .gas_limit(1_000_000_000)
+            .min_auction_period(10)
```

Comment:
seems that this isn't that time sensitive so extra 10s for price discovery seems worth it